src/main/java/uk/gov/hmcts/reform/managecase/service/CaseAssignmentService.java

```diff
@@ -195,8 +197,16 @@ private UserDetails toUserDetails(ProfessionalUser user, List<String> caseRoles)
     }
 
     private CaseDetails getCase(CaseAssignment input) {
-        Optional<CaseDetails> caseOptional = dataStoreRepository.findCaseBy(input.getCaseTypeId(), input.getCaseId());
-        return caseOptional.orElseThrow(() -> new CaseCouldNotBeFetchedException(CASE_COULD_NOT_BE_FETCHED));
+        CaseDetails caseDetails = null;
+        try {
+            caseDetails = dataStoreRepository.findCaseByCaseIdExternalApi(input.getCaseId());
+        } catch (FeignException e) {
```

Comment:
Can this Feign block inside repository ??

Reply:
Resolved in https://github.com/hmcts/aac-manage-case-assignment/pull/173/commits/d691c210e88b0c5d05681a6b8fcd5b123858c829